test/data/gvcf_test_cases/format_fields_integrated.yml

```diff
@@ -119,7 +119,7 @@ truth_output_vcf:
       ##contig=<ID=21,length=48129895>
       #CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	NA12878	NA12879
       21	10009461	.	T	A	.	.	.	GT:AD:DP:GQ:SB:RNC	0/1:10,2:12:16:1,9,1,1:..	0/1:10,2:12:16:1,9,1,1:..
-      21	10009462	.	T	A	0	.	.	GT:DP:GQ:AD:RNC	0/0:14:36:14,0:..	0/1:12:16:10,2:..
-      21	10009463	.	C	G	0	.	.	GT:DP:GQ:AD:RNC	0/0:14:36:14,0:..	0/1:12:16:10,2:..
+      21	10009462	.	T	A	0	.	.	GT:DP:GQ:AD:SB:RNC	0/0:14:36:14,0:.,.,.,.:..	0/1:12:16:10,2:1,9,1,1:..
```

Comment:
VCF spec seems quite ambiguous on this... I would say it's invalid, because there's nothing that suggests it's valid, and so I presume the `NUMBER` specification need to be upheld. The spec only  presented a `NUMBER=1` field. I think, however, it's actually allowed to specify the field in `FORMAT` but not have a value for the trailing fields.

From VCF spec 4.2:
```
If any of the fields is missing, it is replaced with the missing value. For example if the FORMAT is GT:GQ:DP:HQ
then 0 | 0 : . : 23 : 23, 34 indicates that GQ is missing. Trailing fields can be dropped (with the exception of the GT
field, which should always be present if specified in the FORMAT field).
```

Therefore, if we put SB as the last field, it should be droppable in the emission (which will be desirable in keeping output nice, but not very nice for our `diff` comparison).

Implementation-wise, I'm not sure how this is handled... the FORMAT field emission is in large parts a 2 step process:

- Format field update: from [htslib bcf_update_format*](https://github.com/samtools/htslib/blob/b6aa0e6e2350ca7836c275082f1d680fdca9e335/htslib/vcf.h#L581), it seems like we need to pad missing values for every value slot

```
the same number of elements  is expected for each sample. Missing values must be padded with bcf_*_missing or bcf_*_vector_end values.
```

The `bcf_*_vector_end` value seems to be only relevant when describing haploid GTs from [this discussion](https://github.com/samtools/bcftools/issues/227), so I think we're doing the right thing to pad with `bcf_*_missing`.

- Which leaves I think the problem to the actual emission/serialization process on how it handles the `.,.,.,.` represented in memory... This seems to be done at the high level by [this function](https://github.com/samtools/htslib/blob/b6aa0e6e2350ca7836c275082f1d680fdca9e335/vcf.c#L1255), but it seems quite abstracted and I'm not sure how much value we'll gain by trudging through this to get to the actual emission code.

I'm gonna try 1 quick thing of moving RNC to be the first format field (and by so doing making `SB` the trailing field) to see if `htslib` automatically handle it.